download-directory/src/main/resources/config.properties

```diff
@@ -1,9 +1,11 @@
-username=mySQLUsername
-password=mySQLPassword
-database=release_current
-host=localhost
-port=3306
-release=releaseNumber
+release.database.user=releaseDatabaseUsername
+release.database.password=releaseDatabasePassword
+release_current.name=release_current
+release.database.host=releaseDatabaseHost
+release.database.port=3306
+
+releaseNumber=
```

Comment:
Is there a mechanism to sync the release number obtained by Jenkins to be the same as here?

Reply:
That is one of the things handled by the ConfirmReleaseConfigs steps